pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -465,15 +466,13 @@ func getObjectMeta(val *Values, sts *appsv1.StatefulSet) metav1.ObjectMeta {
 
 const (
 	scaleToMultiNodeAnnotationKey = "gardener.cloud/scaled-to-multi-node"
-	ownedByAnnotationKey          = "gardener.cloud/owned-by"
-	ownerTypeAnnotationKey        = "gardener.cloud/owner-type"
 )
 
 func getStsAnnotations(val *Values, sts *appsv1.StatefulSet) map[string]string {
 	annotations := utils.MergeStringMaps(
 		map[string]string{
-			ownedByAnnotationKey:   fmt.Sprintf("%s/%s", val.Namespace, val.Name),
-			ownerTypeAnnotationKey: "etcd",
+			common.GardenerOwnedBy:   fmt.Sprintf("%s/%s", val.Namespace, val.Name),
```

Comment:
These changes are completely unrelated to etcdcopybackuptask and should not be clubbed with this PR. Please remove this and file a separate PR for this.